### PR TITLE
Added Component Content Box

### PR DESCRIPTION
**Pull request by inakijv**
Most important aspects:
- Added Default Content Box and Default Content Box List.
screen shot 2017-08-24 at 11 06 12 pm

### CODE REVIEW

**Comment by inakijv** [comment]
Ping @SylvainGuittard

**Comment by dpobel** [comment]
I don't really see the point in introducing a new component for that. That's visually very close to the table toolbar (if we except the background color). Also given how those toolbars are implemented in Hybrid Platform UI (see [the CSS](https://github.com/ezsystems/hybrid-platform-ui/blob/master/src/bundle/Resources/public/css/modules/list-toolbar.css) and for instance [this template](https://github.com/ezsystems/hybrid-platform-ui/blob/de65dee7aa1f6588bc309c8b48de588688f6fd32/src/bundle/Resources/views/content/tabs/urls.html.twig#L42-L44)) I think this should be a small variation of the list-toolbar (whose name is wrong and should be change as we use it for others things than *list* like here), that way we could get the expected result with the following markup:

```html
<div class="ez-list-toolbar ez-list-toolbar-contentbox">
    <h2 class="ez-list-toolbar-label"></h2>
</div>
```

**Comment by dpobel** [comment]
(erased my previous comment which was wrong)

Hybrid Platform UI has a component *list toolbar* to implement similar UI for tables (see [CSS](https://github.com/ezsystems/hybrid-platform-ui/blob/master/src/bundle/Resources/public/css/modules/list-toolbar.css) and [example](https://github.com/ezsystems/hybrid-platform-ui/blob/de65dee7aa1f6588bc309c8b48de588688f6fd32/src/bundle/Resources/views/content/tabs/urls.html.twig#L42-L44). I think we should reuse that for this component. So basically the markup (btw didn't you forget to commit/push some HTML file?) could be:

```html
<div class="ez-contentbox">
  <div class="ez-list-toolbar">
    <h2 class="ez-list-toolbar-label">Whatever title</h2>
  </div>
  <div class="ez-contentbox-body">
    <p>Some content</p>
  </div>
</div>
```

**Comment by inakijv** [comment]
Updated @dpobel. Thank you for pointing html changes out

**Comment by inakijv** [comment]
Ping @SylvainGuittard

File: css/components/contentbox.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+.ez-contentbox-list .contentbox-value {
+    display: inline-block;
+    width: 66.673%;
+    padding: .75em 0;
+    background-color: #f5f5f5;
+    font-weight: normal;
+    line-height: 22px;
+}
+
+.ez-contentbox-list  .contentbox-value p {
```
Comment (SylvainGuittard) [inline css/components/contentbox.css]: CS: extra space

**Review by SylvainGuittard — CHANGES_REQUESTED** [review]

**Comment by SylvainGuittard** [comment]
+1 besides my comment

**Review by SylvainGuittard — APPROVED** [review]

**Pull request merged**